app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::ReviewsController < ApiController
+
+    def index
+        trail = Trail.find(params[:trail_id])
+        reviews = trail.reviews
+        
+        render json: reviews
+    end
```

Comment:
this unfortunately will all need to be deleted. You should not be fetching the reviews individually when you could just add them to the response of your trails#show. See comment in that controller